changelog.md

```diff
@@ -1,5 +1,10 @@
 # grinder.dart changes
 
+## UNRELEASED
+- Introduce `RunOptions` to support all `Process.run`/`Process.start` 
```

Comment:
Or maybe just re-label the 0.7.1 section as UNRELEASED?  I like the idea of waiting to give a version number to the "next version" until it's ready to be released, since it's hard to know what the next version number will actually be until you're ready to release, since it depends whether you need to include breaking changes, and if you want to do any dev releases.